lib/src/polymer_base.dart

```diff
@@ -229,6 +229,12 @@ abstract class PolymerBase implements CustomElementProxyMixin {
         .apply([path, convertToJs(value), fromAbove], thisArg: jsElement);
   }
 
+  /// Notify that a value at a path has been changed (without passing the value).
+  void notifyPath1(String path) {
```

Comment:
> Should we use some default constant to differentiate those two cases, like this:

Yes, its important to only call the js `notifyPath` function with a single value in this case as well (it changes behavior based on the number of values in the `arguments` variable).